ripe/atlas/tools/commands/probes.py

```diff
@@ -166,6 +166,15 @@ def add_arguments(self):
                 "command."
             )
         )
+        self.parser.add_argument(
+            "--status",
+            type=int,
+            choices=[0, 1, 2, 3],
```

Comment:
Ah well if the API doesn't support multiple values, yeah, let's not do that.  I'd still like to see human-friendly words for this, but I'm happy to do that as a separate PR at some point.